doc/news/_preparation_next_release.md

```diff
@@ -58,14 +58,16 @@ docker run -it elektra/elektra
 ### Session Recording
 
 Elektra now comes with a powerful new feature that allows users to record and export changes made to the KDB: session recording.
-This feature enables you to easily track changes made to the configuration database over time, which helps troubleshoot issues, diagnose errors, and improve system performance.
+This feature enables you to easily track changes made to the configuration database over time, which helps troubleshoot issues, diagnose
```

Comment:
Please avoid such unrelated changes. It also doesn't confirm to our documentation guidelines.

Reply:
Not sure how this happened, but reverted it